pyproject.toml

```diff
@@ -20,17 +20,23 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.8,<3.11"
 numpy = "^1.23.1"
 pandas = "^1.4.3"
-SQLAlchemy = "^1.4.39"
+SQLAlchemy = "^1.4.27"
+mutate = "^0.0.1-alpha.13"
+google-auth = "^2.9.1"
```

Comment:
Dependencies for different backhands should not be mandatory. Instead they should be installed using extras (eg `pip install pydiverse.pipedag[bigquery]`